parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/BrokerPollSystem.java

```diff
@@ -43,6 +43,11 @@
 
     private Optional<Future<Boolean>> pollControlThreadFuture;
 
+    /**
+     * While {@link io.confluent.parallelconsumer.internal.State#paused paused} is an externally controlled state that
+     * temporarily stops polling and work registration, the {@code paused} flag is used internally to pause subscriptions
+     * if polling needs to be throttled.
+     */
     @Getter
     private volatile boolean paused = false;
```

Comment:
ename paused 
```suggestion
    private volatile boolean subscriptionsShouldBePaused = false;
```

Reply:
Will do

Reply:
Obsolete as all changes to `BrokerPollSystem` have been reverted.